components/auth/AcceptableUse.tsx

```diff
@@ -59,7 +62,7 @@ export const AcceptableUseTerms = (props: AcceptableUseProps): React.ReactElemen
         <h1>{t("acceptableUsePage.welcome")}</h1>
         <span>
           <>
-            {t("acceptableUsePage.lastLoginTime")} : {lastLoginTime}
+            {t("acceptableUsePage.lastLoginTime")} : {time}
```

Comment:
If `time` is undefined, should this <span> even be displayed?

Reply:
```
{ time && (
  <span>
    <>
      {t("acceptableUsePage.lastLoginTime")} : {time}
    </>
  </span>
  )}
  ```

Reply:
You're right and i've thought about it but I'm not sure if that was the requirement. 
Either way showing  the label may quickly reveal that the feature is broken (visually).